jenkins/dev_env/conf/example_config_hw.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
there are two file single and multi vm config json

Please update those

we not need two file for hw and vm
update previous file

Reply:
Earlier I was updated the same files but Ajay S. comments was to create separate files for VM & HW. hence created separate files.

Reply:
After speaking with Ajay P, Updated the things in same files only and added ENV_TYPE in dev conf file